### DISCUSSION TITLE
Ava reporters

### DISCUSSION THREAD

**Discussion by akash10anand** [avajs/ava#2633] [Q&A] [answered]
Are there any existing reporters for AVA ? Any html or json reporters ?

**Comment by sindresorhus** [comment, marked as answer]
You can use any TAP reporter: https://github.com/avajs/ava/blob/master/docs/05-command-line.md#tap-reporter

See: https://github.com/sindresorhus/awesome-tap#javascript

For example: https://github.com/gummesson/tap-json

**Reply by akash10anand** [reply]
Thanks.
